src/style.css

```diff
@@ -1 +1,57 @@
-/* Styles go here */
+* {
+  margin: initial;
+}
+
+html {
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+body {
+  background: #e5e5e5;
+}
+
+.menu {
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 40px 0 50px;
```

Comment:
paddings should be similar on both sides. Better remove the right padding from the last link